test/test_helper.exs

```diff
@@ -19,6 +22,10 @@ defmodule TestHarness do
     end
   end
 
+  defp check_results(results) do
+    Enum.each results, &(assert &1 == :passed)
```

Comment:
I can see the appeal... for some reason I've always been cautious with the parens.
@ukutaht What's your take?